src/js/components/CombinationCanvas.vue

```diff
@@ -101,6 +101,7 @@ export default {
                 this.artefacts.forEach(artefact => {
                     artefact.rect = geoJsonParseRect(artefact.rect)
                     artefact.poly = geoJsonPolygonToSvg(artefact.poly, artefact.rect)
+                    console.log(artefact.poly)
```

Comment:
@Bronson-Brown-deVost : Was this something to help debugging/development? Should it be committed?

Reply:
Ha, I should be committed.  That was to get some data to use in my testing unit.  It should be removed, nice catch.